jobs/service-fabrik-broker/templates/bin/drain.erb

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
just to correct my understanding from https://bosh.io/docs/drain.html , this script will get executed whenever the broker process is restarted or stopped correct ?

In case we try to manually play with the monit process after this change , this will execute correct and we should be aware of the maintenance state while we do such manual activity ?

Reply:
I dont think so, atleast prestart, poststart documentation explicitly states this that monit start/stop does not invoke them. So i believe this is true for even drain. Also no harm will be done in rerunning drain as it will not update maintenance if system is already not in it. So it just comes out, if system is not in maintenance. Note that drain is not putting system in maint. It will only update the state of maintenance if someone has already started it. 
Similar to the 'supported' transaction isolation level:-)

Reply:
Yes agree.